### PR TITLE
Add italian language

### PR DESCRIPTION
**Pull request by maxcanna**
Add italian localization

### CODE REVIEW

File: app/i18n/it-IT.po [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+msgstr "N/D"
+
+msgid "on"
+msgstr "ON"
+
+msgid "off"
+msgstr "OFF"
+
+msgid "exe"
+msgstr "ESEGUI"
```
Comment (smirko-dev) [inline app/i18n/it-IT.po:12]: @maxcanna Does that text fit into the UI element? Did you test that (e.g. with Versa 2)?

Comment (maxcanna) [reply app/i18n/it-IT.po]: > @maxcanna Does that text fit into the UI element? Did you test that (e.g. with Versa 2)?

Yes, you're right.

What about using ▶️ ?
Take a look:

image

Comment (smirko-dev) [reply app/i18n/it-IT.po]: Is there no understandable way to express run or execute in Italian with just 3 letters?
If not, than the play icon is a good alternative solution.

**Comment by maxcanna** [comment]
> I added ellipsis to the end of friendly name otherwise it overlaps with the button:
> 
> image

Oh, I forgot to make the same thing for sdk 5 🤦🏻‍♂️

File: app/i18n/it-IT.po (hunk 1 above)
Comment (maxcanna) [reply app/i18n/it-IT.po]: Yes I can truncate it to _ESE_ if you want to stay coherent

Comment (smirko-dev) [reply app/i18n/it-IT.po]: Yes, truncation would be perfect

Comment (maxcanna) [reply app/i18n/it-IT.po]: ok, done

File: resources/styles.sdk4.css [hunk 2]
```diff
@@ -19,7 +19,9 @@
   font-family: Colfax-Regular;
   text-anchor: start;
   text-length: 64;
-  fill: #FFFFFF
+  fill: #FFFFFF;
+  overflow: hidden;
+  text-overflow: ellipsis;
```
Comment (smirko-dev) [inline resources/styles.sdk4.css:24]: Can you please change the styles.sdk5.css as well?

Comment (maxcanna) [reply resources/styles.sdk4.css:24]: Oh right! 🤦🏻‍♂️ 😁

Comment (maxcanna) [reply resources/styles.sdk4.css:24]: Done

**Pull request merged**